mail_layout_preview/readme/CREDITS.md

```diff
@@ -0,0 +1,10 @@
+The development and migration of this module from 15.0 to 16.0 has been
+financially supported by:
+
+- Camptocamp
+
+
```

Comment:
```suggestion
```
remove it

Reply:
why remove traces from the past? 

> The development and migration of this module from 15.0 to 16.0 and from 17.0 to 18.0 has been
financially supported by:

Reply:
I thought when he append message `17.0 to 18.0`, we don't need old message above